storage/src/journal/fixed.rs

```diff
@@ -611,7 +653,7 @@ mod tests {
             let blob = context
                 .open(&cfg.partition, &40u64.to_be_bytes())
                 .await
-                .expect("Failed to open blob");
+                .expect("Failed to remove blob");
```

Comment:
should this still be `open`? I also think a `\n` after this.

Reply:
yeah weird not sure why I changed that, reverted.